### PR TITLE
[processors] updating remaining processor scope names

### PR DESCRIPTION
**Pull request by codeboten**
Doing this in a single PR as there are no additional changes other than running the generation.

Part of open-telemetry/opentelemetry-collector#9494

### CODE REVIEW

**Review by crobert-1 — COMMENTED** [review]
Looks like the issue in each changelog entry needs to be updated to this PR

File: .chloggen/codeboten_update-scope-deltatocumulativeprocessor.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: deltatocumulativeprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the deltatocumulativeprocessor from `otelcol/deltatocumulative` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-deltatocumulativeprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-filterprocessor.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: filterprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the filterprocessor from `otelcol/filter` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-filterprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-groupbyattrsprocessor.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: groupbyattrsprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the groupbyattrsprocessor from `otelcol/groupbyattrs` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbyattrsprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-groupbyattrsprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-groupbytraceprocessor.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: groupbytraceprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the groupbytraceprocessor from `otelcol/groupbytrace` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbytraceprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-groupbytraceprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-k8sattributesprocessor.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: k8sattributesprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the k8sattributesprocessor from `otelcol/k8sattributes` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-k8sattributesprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-probabilisticsamplerprocessor.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: probabilisticsamplerprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the probabilisticsamplerprocessor from `otelcol/probabilisticsampler` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-probabilisticsamplerprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-routingprocessor.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: routingprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the routingprocessor from `otelcol/routing` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-routingprocessor.yaml:13]:
```suggestion
issues: [34550]
```

File: .chloggen/codeboten_update-scope-tailsamplingprocessor.yaml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: tailsamplingprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Update the scope name for telemetry produced by the tailsamplingprocessor from `otelcol/tailsampling` to `github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor`"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34429]
```
Comment (codeboten) [inline .chloggen/codeboten_update-scope-tailsamplingprocessor.yaml:13]:
```suggestion
issues: [34550]
```

**Review by codeboten — COMMENTED** [review]
🤦🏻

**Review by crobert-1 — APPROVED** [review]

**Review by mwear — APPROVED** [review]

**Review by RichieSams — APPROVED** [review]

**Pull request merged**